src/generator/backend/assets/js/popover.js

```diff
@@ -1,7 +1,8 @@
 /* Created by aayusharora on June 20, 2017 */
 
 $(document).ready(function() {
-  $('.pop-box').hide();
+  var $popbox = $('.pop-box');
```

Comment:
just call is `popbox` and not `$popbox` then ? 
$() is for jQuery stuff only. If it's our on var, just name it simple in letters.